core/templates/dev/head/app.js

```diff
@@ -195,6 +195,15 @@ oppia.factory('oppiaDatetimeFormatter', [function() {
         return date.toLocaleTimeString().replace(/:\d\d /, ' ');
       }
       return date.toLocaleDateString();
+    },
+    // Returns just the date.
+    getLocaleAbbreviatedDateString: function(millisSinceEpoch) {
+      var date = new Date(millisSinceEpoch);
+      if (date.toLocaleDateString() == new Date().toLocaleDateString()) {
+        // If the date is the same, return current date.
+        return new Date().toLocaleDateString();
+      }
+      return date.toLocaleDateString;
```

Comment:
Tests in appSpec.js?

Looking through our emails, I see you want tests to make sure everyone sees the same contribution date, regardless of who they're logged in as. Are these the tests you are referring to? These tests would fall under profile_test.py, correct?